components/inputs/input-number.js

```diff
@@ -95,6 +95,13 @@ class InputNumber extends FocusMixin(LabelledMixin(SkeletonMixin(FormElementMixi
 			 * @type {boolean}
 			 */
 			disabled: { type: Boolean },
+			/**
+			 * ACCESSIBILITY: Hidden label, different from `label`, to be used on the input to provide more context for screenreader users.
+			 * To be used only if `label` is defined and `label-hidden` is NOT used.
+			 * Note that screenreader users will likely NOT hear the `label`.
+			 * @type {string}
+			 */
+			hiddenLabel: { attribute: 'hidden-label', type: String },
```

Comment:
In `input-text` this is `aria-label`. We have more recently been trying to avoid `aria-*` attributes since those should generally only be used on the appropriate html elements (and also can cause axe errors).

For that reason I gave this a different name. I'd also understand wanting to keep `aria-label` since that's what it's passed through to `input-text` on (though perhaps that one should also end up renamed - I believe we have an epic to get more consistency here). Also, I don't know if `hiddenLabel` is good since `labelHidden` is also a property here.

@geurts curious your thoughts here as well

Reply:
What if we started a pattern of using the property (i.e. `.ariaLabel="whatever"`), make sure they don't reflect, and then we could pass the value through to the `input`.

Reply:
I like the idea of starting a more defined pattern for passing through aria properties when necessary. The dot prefix works nicely. Rather than camel case, could it just be formatted the same way the standard attribute is? So it would be `.aria-label`?

Reply:
No, the `.` is a directive to reference the JavaScript property instead of the HTML attribute. As such, the consumer would be referencing the JavaScript property `ariaLabel` (as defined by the browser in this case). 

I generally feel we should be keeping the `aria-*` attributes internal when they are limited to the internal implementation (like we do for `description`). Imo, we should not leak internal implementation details on the host. If the host is focusable and has a meaningful role, then it's fair game to use aria attributes on them, but they should not be treated as pass-through attributes.

Yesterday I saw someone suggesting/proposing (in W3C forums) support for `role="none"` on custom elements, so that such `aria-*` properties would be more like "pass-through" attributes. Not sure what the outcome of that discussion will be, but it's a topic on others' radar.

We have a lot ways to render labels & descriptions - the `inline-help` slot (rendered below), `instructions` (rendered in tooltip), and `description` (render offscreen with `aria-describedby`). I worry a bit about adding another to the mix. Is there a reason `description` doesn't suffice for this?

Reply:
Agree with Dave here on the `aria-*` stuff -- we want to avoid it. It's already pretty confusing with `label` (visual), `label-hidden` (takes the visual label and moves it to `aria-label`) plus the other things Dave mentioned. The discussion in Slack is continuing to the point where I don't think we currently need this anymore anyway.